src/Console/Command/FixCommand.php

```diff
@@ -159,7 +162,7 @@ public function getHelp(): string
 
                 <info>$ php %command.full_name% --verbose --show-progress=dots</info>
 
-            By using <command>--using-cache</command> option with `yes` or `no` you can set if the caching
+            By using <comment>--using-cache</comment> option with `yes` or `no` you can set if the caching
```

Comment:
myself, i would make independent PR , that could get merged earlier.
(but no worry, no need to extract now)

Reply:
Yeah, it would be better because it does not belong here, but making PR for that would be too much probably 😅.